### ISSUE TITLE
Visualization of training process in keras

### ISSUE THREAD

**Issue by CSCSCS111** [zhixuhao/unet#81]
What can be done to visualize the training process in keras, such as acc, loss, etc.I am doing a research and study on my training set. I really hope to get guidance. Thank you very much.
